doc/urgency.md

```diff
@@ -0,0 +1,12 @@
+# Urgency
+
+Urgency is pretty primitive at the moment. It should only affect tasks that are
+not resolved.
+
+It takes the Task's priority as a Fibonacci (Low = 1, Critical = 5) and an applies a multiplier (5).
```

Comment:
this is very difficult for a layman (including myself) to understand.

Reply:
Agreed, and the reasoning behind it doesn't come naturally. 

I suggest an approach of identifying then addressing individual problems that the current sorting method has; note that if the basic method is good 95% it's probably a good solution

Reply:
This was a complete guess to start out.  I left the implementation mostly as a way to start the conversation about what the weights should be for everything.